src/components/Login/LoginMobileNumberCard.tsx

```diff
@@ -68,9 +71,15 @@ export const LoginMobileNumberCard: FunctionComponent<LoginMobileNumberCard> = (
 
   const onSubmitMobileNumber = (): void => {
     if (!countryCodeValidator(countryCode)) {
-      alert("Invalid country code");
+      showAlert({
+        ...wrongFormatAlertProps,
+        description: "Invalid country code"
+      });
     } else if (!mobileNumberValidator(countryCode, mobileNumberValue)) {
-      alert("Invalid mobile phone number");
+      showAlert({
+        ...wrongFormatAlertProps,
+        description: "Invalid mobile phone number"
```

Comment:
Hmm, ERROR_MESSAGE.INVALID_PHONE_NUMBER's message is "Enter valid contact number". 

Should we change mobile phone number to contact number? or leave it as is haha

Reply:
yes let's standardise to contact number!